src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -248,37 +234,6 @@ export function validateData(limits: IChartLimits, chartOptions: IChartOptions):
     };
 }
 
-export function isDerivedMeasure(measureItem: Execution.IMeasureHeaderItem, afm: AFM.IAfm) {
-    return afm.measures.some((measure: AFM.IMeasure) => {
-        const measureDefinition =
-            get(measure, "definition.popMeasure") || get(measure, "definition.previousPeriodMeasure");
-        const derivedMeasureIdentifier = measureDefinition ? measure.localIdentifier : null;
-        return (
-            derivedMeasureIdentifier &&
-            derivedMeasureIdentifier === measureItem.measureHeaderItem.localIdentifier
-        );
-    });
-}
-
-function findMeasureIndex(afm: AFM.IAfm, measureIdentifier: string): number {
-    return afm.measures.findIndex((measure: AFM.IMeasure) => measure.localIdentifier === measureIdentifier);
-}
-
-export function findParentMeasureIndex(afm: AFM.IAfm, measureItemIndex: number): number {
```

Comment:
I think you should move this to the place where it is used. Because stuff you move to utils should be tested. And it is used once anyway